### DISCUSSION TITLE
Add Personal Protective Equipment (PPE) needs.

### DISCUSSION THREAD

**Discussion by athenaopp** [neherlab/covid19_scenarios#522] [General]
Could you add the  PPEs requirements to the model? That would allow jurisdictions and hospitals to have a good estimate/forecast their PPEs Needs. I think you would need to add:
1. Number of hospitalized persons (not admitted critical care)
2. Number of critical care/intensive care (to include in hospital but not critical care and in home/community) 
Include a field for assumed number of days in care and staff PPE requirements needs (see below)

(Later revision could expand to include first responders, facilities like jails/nursing homes and at home). 

Here is a link for the estimates of PPEs needed per staff member (based on Ebola: couldn’t find one for Covid yet) https://www.cdc.gov/vhf/ebola/healthcare-us/ppe/calculator.html

Here is WHO’s lost of staff/ scenarios PPE requirements: https://apps.who.int/iris/bitstream/handle/10665/331498/WHO-2019-nCoV-IPCPPE_use-2020.2-eng.pdf

**Comment by ivan-aksamentov** [comment]
@athenaopp Thanks!
This sounds like something very useful and there's been a discussion previously in this issue: https://github.com/neherlab/covid19_scenarios/issues/162
citing another calculator, however we did not end up with anything particular.

We don't have that much resources so if you'd like to actively participate - that would definitely improve the chances of this feature being implemented.

cc @rneher @nataliadgepi @Johan-A-M

**Reply by athenaopp** [reply]
Yeah that’s something I could get done. What would it need to look like to be useful (sorry new to this community)? How about an excel file with the categories of workers and PPE requirements for various scenarios (hospitalization, critical care) and the min and max values? (So per patient scenario per day/per worker counts; Including max and min). I could also include all the definitions in a separate sheet on the workbook. 
cc @rneher @nataliadgepi @Johan-A-M

**Reply by nnoll** [reply]
We could optionally compute the PPE resources needed based on our predicted hospitalization rates within the export button. This is outside my area of expertise - would you expect the PPE availability to dramatically affect outcomes of COVID-19 patients? If not, I think it would suffice to treat this as an optional computation on the already computed output trajectories.

**Reply by athenaopp** [reply]
@nnoll Yes, I think so. The supply of PPEs effects the infection of medical staff and their capacity to effect other people in the hospital/ community. Health care workers representing ~15% cases in various countries (source:  https://www.google.ca/amp/s/www.nytimes.com/2020/03/24/world/europe/coronavirus-europe-covid-19.amp.html). Therefore the supply of PPE can influence not only the rise of cases AND the number of deaths because medical staff are interacting with other sick/vulnerable patients (side note: I don’t think we know at this time the rate of infection of health care workers by PPE scenario yet... I’ll keep a look out). 

This feature would be HUGE tangible benefit to hospitals and governments because with the limited international supply of PPEs, it is difficult to accurately predict how many PPEs they need to keep everyone safe. 

cc @rneher @nataliadgepi @Johan-A-M

**Comment by codycotner** [comment]
Hi everyone,

I work with the University of Pennsylvania's CHIME team, and we've created a PPE calculator that can be used by any model that forecasts the number of hospitalized patients. I had a meeting with the CHIME team today, and we would be interested in collaborating. Let us know if incorporating our PPE model into your forecast would be of interest. Happy to send an introductory email to the rest of the team if you are interested. Thanks.

**Comment by rneher** [comment]
A simple addition would be to add columns to the table export. If the PPE demand can be calculated from the trajectories for ICU and hospitalized patients, this could be pretty straightforward. Adding more stuff to the plots would clutter them too much though. 

@codycotner happy to be connected and see whether there are any good ways to use each others models/calculators!

**Reply by codycotner** [reply]
Sounds great - could you send an email to cody.cotner@pennmedicine.upenn.edu and I will connect you to the folks leading the effort?

**Reply by nnoll** [reply]
I opt for this computation as I can't think of a generic enough way to integrate PPE usage/availability/limitations into our model in a simple way.

**Reply by athenaopp** [reply]
Yeah I would be very interested in helping any way I can. Let me know what I can to do to support the initiative :)
